engines/griffon/cutscenes.cpp

```diff
@@ -182,6 +186,20 @@ void GriffonEngine::showLogos() {
 }
 
 void GriffonEngine::intro() {
+	#ifdef USE_TTS
+		Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+		const char *story1 = story[0];
+		char result[1000];
+
+		strcpy(result,story1);
```

Comment:
This could really need some buffer overflow checks - or use Common::String and the append operator for this.